freecodecamp.conf.json

```diff
@@ -47,6 +48,16 @@
       "english": "./curriculum/locales/english"
     }
   },
+  "hotReload": {
```

Comment:
does the client hot reload when I save files I create to do a project?

Reply:
The client does not hotReload at all. This controls the `hotReload` function in `.freeCodeCamp/tooling/hot-reload.js`, and is to list the files ignored when running the tests on watch.